ts/sagas/startup/checkSupportedFingerprintSaga.ts

```diff
@@ -11,12 +11,10 @@ export type BiometrySimpleType =
   | "Fingerprint"
   | "FaceID"
   | "TouchID"
-  | "Not Enrolled"
+  | "NotEnrolled"
   | "Unavailable";
 
 export function* checkSupportedFingerprintSaga(): IterableIterator<Effect> {
-  // We check whether the user has already created a PIN by trying to retrieve
```

Comment:
comments are needed to understand what's going on, don't remove them, just write the correct sentence

Reply:
restored them in 2fb4d8ee